process/process.go

```diff
@@ -163,6 +164,7 @@ func NewProcess(pid int32) (*Process, error) {
 	if !exists {
 		return p, ErrorProcessNotRunning
 	}
+	go p.CreateTime()
```

Comment:
If we use `go` here, create time becomes unstable. How about use `CreateTimeWithContext()` with `WithTimeout()` ?

Reply:
I chose to spawn a goroutine so `NewProcess` performances wouldn't be impacted by this new call to `CreateTime`. I don't see how create time gets unstable with this call as it is then cached after first call.

Reply:
hmmm. it depends on running environment. but I can understand. Thanks.